ledger-core/virtual/execute/execute.go

```diff
@@ -541,21 +535,38 @@ func (s *SMExecute) getDescriptor(state *object.SharedState) descriptor.Object {
 func (s *SMExecute) stepStartRequestProcessing(ctx smachine.ExecutionContext) smachine.StateUpdate {
 	var (
 		objectDescriptor descriptor.Object
+		isDeactivated    bool
 	)
 	action := func(state *object.SharedState) {
+		objectDescriptor = s.getDescriptor(state)
+		if objectDescriptor != nil && objectDescriptor.Deactivated() {
+			if !state.KnownRequests.Delete(s.execution.Isolation.Interference, s.execution.Outgoing) {
```

Comment:
this is not correct way. why do you Delete anything from the table?

Reply:
Previously, we only added this reference, that is why I only remove this reference

Reply:
What do you mean 'anything'?

Reply:
I remove 'Delete'. And we store result of such execution